mm2src/coins/lp_coins.rs

```diff
@@ -821,17 +836,19 @@ impl WithdrawRequest {
             amount,
             max,
             fee,
+            memo,
         }
     }
 
-    pub fn new_max(coin: String, to: String) -> WithdrawRequest {
+    pub fn new_max(coin: String, to: String, memo: Option<String>) -> WithdrawRequest {
```

Comment:
`new_max` should have same behaviour with `new` with setting the amount to max. I think it shouldn't set any different values by default.

Reply:
Btw, `WithdrawRequest::new` is not used. IMO, it's better to initialize a structure with more than 3 fields straightforward instead of using a constructor that takes all of the properties. So
```rust
let req = WithdrawRequest {
  coin: "",
  from: Some(""),
  ..
};
```
What do you think about removing `WithdrawRequest::new` and removing `memo` from `WithdrawRequest::new_max`?

Reply:
Hmm, after removing `new` function, should we keep the other one as `new_max` or rename it to`max_amount`?

Reply:
I think `new_max` is ok. `max_amount` can confuse, it may seem that `max_amount` returns a max available amount for withdraw.

Reply:
Done